map_server/src/image_loader.cpp

```diff
@@ -54,6 +57,34 @@ namespace map_server
 void
 loadMapFromFile(nav_msgs::GetMap::Response* resp,
                 const char* fname, double res, bool negate,
+                double occ_th, double free_th, double* origin)
+{
+  loadMapFromFile(resp,
+                  fname, res, negate,
+                  occ_th, free_th, origin,
+                  TRINARY);
+}
+
+void
+loadMapFromFile(nav_msgs::GetMap::Response* resp,
+                const char* fname, double res, bool negate,
+                double occ_th, double free_th, double* origin,
+                bool trinary)
+{
+  ROS_WARN(
+    "map_server::loadMapFromFile with trinary as bool has been deprecated.",
+    "Please update your code to use MapMode enumerable."
+  );
+  MapMode mode = (trinary)?TRINARY:RAW;
```

Comment:
I think this needs to be SCALE instead of RAW. The RAW option did not exist before the MapMode became an enum.